add-ons/updating/tb78/convert-legacy.md

```diff
@@ -4,76 +4,63 @@ This guide is intended to help developers to port their Legacy WebExtensions to
+As experiments usually run in the main process and have unrestricted access to any aspect of Thunderbird, they are expected to require updates for each new version of Thunderbird. To reduce the maintainance burden in the future, it is in your own interest to use experimental APIs only to the extent necessary for the add-on.
 
-Full examples for [a simple function with parent and child implementations](https://firefox-source-docs.mozilla.org/toolkit/components/extensions/webextensions/functions.html) and [events add-ons can listen for](https://firefox-source-docs.mozilla.org/toolkit/components/extensions/webextensions/events.html) are available in the Firefox source documentation.
+Best practice: Try to write APIs that would be useful for a wide range of add-ons, not just the one you're porting. That way, you can later on propose the API you designed for inclusion in Thunderbird, with your add-on serving as reference implementation. If your APIs become a part of Thunderbird, you no longer need to maintain them as part of the add-on.
 
-If your experiment is so complex that it does not reasonably fit into a single source file, you can use JavaScript modules just like in legacy extensions with some additional boilerplate:
-```javascript
-const { ExtensionParent } = ChromeUtils.import(
-    "resource://gre/modules/ExtensionParent.jsm");
-const extension = ExtensionParent.GlobalManager.getExtension(
-    "insert-your-extension-id-here@example.com");
-const { /* ... exported symbols ... */ } =
-    ChromeUtils.import(extension.rootURI.resolve("path/to/module.jsm"));
-
-// when unloading: (safe to call even if the import is conditional / elsewhere!)
-Components.utils.unload(extension.rootURI.resolve("path/to/module.jsm"));
-```
-
-As experiments usually run in the main process and have unrestricted access to any aspect of Thunderbird, they are expected to require updates for each new version of Thunderbird. To reduce the maintainance burden in the future, it is in your own interest to use experiments only to the extent necessary for the add-on.
-
-Best practice: try to write APIs that would be useful for a wide range of add-ons, not just the one you're porting. That way, you can later on propose the API you designed for inclusion in Thunderbird, with your add-on serving as reference implementation. If your APIs become a part of Thunderbird, you no longer need to maintain them as part of the add-on.
+A more thorough description of experimental APIs can be found in a [separate article](experiments.md).
 
 
 ## Replacing options
 
 Option windows are replaced by [settings pages](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Implement_a_settings_page) defined in `manifest.json`:
+
 ```json
 "options_ui": {
   "page": "options.html"
-}
+},
```

Comment:
Adding a `,` makes the block invalid JSON (I thus removed it in my last revision). Was there a reason for re-adding it?

Reply:
I had a merge conflict and manually pasted changes.I missed this one.